### ISSUE TITLE
Review Medallia survey data to prepare for TE launch

### ISSUE THREAD

**Issue by evelynhilbrichdavis** [department-of-veterans-affairs/va.gov-team#88714]
## Issue Description
Veterans [regularly leave feedback via Medallia surveys](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/accessing-and-utilizing-contact-center-data#AccessingandUtilizingContactCenterData-WhattoDoWiththeData) when they are filing for disability compensation. This ticket captures the work of requesting and reporting on the most recent month of qual/quant data for Intercept and Feedback surveys in the month leading up to the Toxic Exposure launch. 

## Issue Impact
1. This benchmark will help us distinguish ongoing UX challenges from those that will result from the TE rollout in late August
2. Looking at July 2024 Medallia data will give us insight into Veteran thinking/concern leading up to Pact Act deadline.

---
## Tasks
- [X] Request data from Contact Center
- [X] Ask Debt team about their approach for expediting data cleaning and review
- [X] Remove PII and clean data
- [X] Synthesize 
- [] Create a lightweight report and template that will be used for future requests

## Acceptance Criteria
- [ ] Research insights are documented in a lightweight report
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by evelynhilbrichdavis** [comment]
Quantitative outputs from the two Medallia surveys include the following likert scale questions. Historically we've gotten more responses from the Feedback survey (#1 below). This is feedback we'll track now so we can see if there are any changes during the TE release.

**1. Satisfaction (Feedback survey)**
    - What is your overall satisfaction with this site?
    
**2. Overall experience (Intercept survey)**
    - Please rate your experience today
    
**3. Understandability (Intercept survey)**
    - Please rate how understandable this site's information is.
    
**4. Trust (Intercept survey)**
    - How did this interaction change your trust in Veteran Affairs?
    - How much do you trust VA to fulfill our country's commitment to Veterans?

**Comment by shannonkford** [comment]
CSAT in the portfolio is likely to be measured from the intercept (your Q2 above), but I am still waiting on some final feedback from the OCTO team. We'll have that historical data and don't need you to track that separately. It varies quite a bit month to month, so if you want to track the equivalent from the feedback survey for this purpose, you'll need to look at that for more than just the month before to see how much it varies. I think you can monitor qual/open text feedback from the prior month, though.

**Comment by evelynhilbrichdavis** [comment]
Thanks @shannonkford. We'll look back farther for Q1 from Feedback. If I remember correctly we can pull it from the Domo dashboard-- but I'll double check. 
I just cleaned the PII from the Medallia data for July 1-August 9 and will be reviewing that time frame for open text feedback.

**Comment by evelynhilbrichdavis** [comment]
[Github report is in progress. ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-08-Toxic-Exposure/2024-08-Medallia-Review.md)

**Comment by evelynhilbrichdavis** [comment]
Building template for monitoring in this [Github doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-08-Toxic-Exposure/TE-UX-Monitoring-Template.md)

**Comment by evelynhilbrichdavis** [comment]
**UX Monitoring meeting on 8/20 Action Items**

**Notes for OCTO**
- Share out report template and schedule a walkthrough (?)
- Set list of stakeholders to include in the weekly reports
- 
**Questions for the Contact Center**
- Do Veterans have to be logged in to leave Feedback/Intercept responses?
- We found that the Intercept question is firing at the start of the form. Do we have the authority to move it to the end of the form so it only pops up for 5% of users on the confirmation page? (We believe Debt Resolution team had already done this)
- Does Medallia track the user uuid? Is there a datadog event for Medallia that gets captured?
- How is "trust" defined?

**Comment by evelynhilbrichdavis** [comment]
Closing this ticket since Medallia review is finished. Moving attention to TE Monitoring as a whole.
